src/main/java/org/testng/util/TimeUtils.java

```diff
@@ -14,11 +16,26 @@ private TimeUtils() {
      * @param timeInMilliSeconds - The time in milliseconds
      * @param format             - A format that can be used by {@link SimpleDateFormat}
      * @return - A formatted string representation of the time in UTC/GMT timezone.
+     * @deprecated - Method stands deprecated as of TestNG 7.0.0
      */
+    @Deprecated
     public static String timeInUTC(long timeInMilliSeconds, String format) {
         SimpleDateFormat sdf = new SimpleDateFormat(format);
         TimeZone utc = TimeZone.getTimeZone("UTC");
         sdf.setTimeZone(utc);
         return sdf.format(timeInMilliSeconds);
     }
+
+    /**
+     * @param timeInMilliSeconds - The time in milliseconds
+     * @param format             - A format that can be used by {@link SimpleDateFormat}
+     * @return - A formatted string representation of the time in the timezone as obtained via
+     * {@link RuntimeBehavior#getTimeZone()}
+     */
+    public static String formatTimeInLocalOrSpecifiedTimeZone(long timeInMilliSeconds, String format) {
```

Comment:
Can you move the methods to an internal sub package?

Reply:
@juherr - Since this method doesn't expose anything in terms of TestNG internals, I thought of housing this method in the same existing class itself.